content/technotes/2018-08-08-phylotar-release.md

```diff
@@ -0,0 +1,173 @@
+
+```r
+devtools::install_github(repo = 'ropensci/phylotaR', build_vignettes = TRUE)
+```
+
+### BLAST+
+
+In addition to the R package you will also need to have installed BLAST+ -- a local copy of the well-known [BLAST software](https://en.wikipedia.org/wiki/BLAST) on your machine. Unfortnately this can be a little complex as its installation depends on your operating system (Windows, Mac or Linux). Fortunately, NCBI provides detailed installation instructions for each flavour of operating system.
+
+**To install BLAST+ on your local machine follow the [NCBI installation instructions](https://www.ncbi.nlm.nih.gov/books/NBK279671/).**
+
+Once BLAST+ is intalled you will need to record the location of the BLAST+ file system path where the exectuable programs are located. This should be something like `C:\users\tao\desktop\blast-2.2.29+\bin\` on a Windows machine or `/usr/local/ncbi/blast/bin/` on a Unix.
+
+## Quick guide
+
+### Taxonomic ID
+
+After installation, a phylotaR pipeline can be launched with the `setup()` and `run()` functions. Before launching a run, a user must first look up the NCBI taxonomic ID for their chosen taxonomic group. To find the ID of your taxon, search [NCBI taxonomy](https://www.ncbi.nlm.nih.gov/taxonomy) using the scientific name or an English common name. For example, upon searching for the primate genus 'Aotus' we should come across this [page](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=9504) which states the taxonomic ID as `9504`.
+
+Because names are not unique phylotaR does not allow names to be used instead of taxonomic IDs. Aotus is a good example as there is also a plant genus of the same name. If we were to search NCBI with just the name then we would be downloading a mixture of primate and plant DNA. Such is the importance of IDs. (Alternatively, you could automate this step yourself using the [taxize](https://github.com/ropensci/taxize) package.)
+
+### Running
+
+With the package installed, BLAST+ available and ID to hand, we can run our pipeline.
+
+```r
+# load phylotaR
+library(phylotaR)
+# create a folder to hold the cached data and results
+dir.create('aotus')
+# setup the folder and check that BLAST is available
+setup(wd = wd, txid = 9504, ncbi_dr = '/usr/local/ncbi/blast/bin/')
+# run the pipeline
+run(wd = wd)
+# ^^ running should take about 5 minutes to complete
+```
+
+For more details on running the pipeline, such as changing the parameters or understanding the results, see the vignette, `vignette('phylotaR')` or visit the [website](https://ropensci.github.io/phylotaR/).
+
+### Timings
+
+The time it takes for a pipeline to complete depends on the number of taxonomic groups a taxon contains and the number of sequences it represents. The genus Aotus has more species than the great apes but the latter will take far long because it represents highly well-sequenced species.
```

Comment:
> The genus Aotus has more species than the great apes but the latter will take far long because it represents highly well-sequenced species.

It took me some time to understand this sentence, partly because there is no entry in table for "great apes". You can probably omit the sentence since it leads the reader's attention away